File: Daily_Practice/151_Reverse_Words_In_a_string.go
package main

import (
	"fmt"
	"strings"
)

func ReverseWords(st string) string {
	sl := strings.Split(st, " ")
	var reverse = []string{}
	for i := len(sl) - 1; i >= 0; i-- {
		reverse = append(reverse, sl[i])
	}
	output := strings.Join(reverse, " ")
	return output
}

func main() {
	st := "My name is kartik"
	op := ReverseWords(st)
	fmt.Println(op)
}

File: Daily_Practice/151_Reverse_Words_In_a_String.py
def ReverseWords(st: str)->str:
    li = list(st.split(" "))
    temp = li[::-1]
    reverse = " ".join(temp)
    reverse = " ".join(reverse.split())
    return reverse

st = "My name is     kartik"
op = ReverseWords(st)
print(op)